BriefAssistant/Controllers/BriefsController.cs

```diff
@@ -191,7 +191,23 @@ public BriefsController(IHostingEnvironment env, IEmailSender emailSender, Appli
             return await _applicationContext.Briefs
                 .Include(brief => brief.ContactInfoDto)
                 .Include(brief => brief.CircuitCourtCaseDto)
-                .SingleAsync(brief => brief.Id == id);
+                .SingleOrDefaultAsync(brief => brief.Id == id);
+        }
+
+        [HttpPost("{id}/delete")]
+        public async Task<IActionResult> Delete(Guid id)
```

Comment:
Method is missing authentication. The way it is currently setup anyone can delete any brief if they know the id